addons/web/static/src/views/form/form_controller.js

```diff
@@ -301,11 +316,18 @@ export class FormController extends Component {
                 key: "delete",
                 description: this.env._t("Delete"),
                 callback: () => this.deleteRecord(),
+                isDeleteAction: true,
```

Comment:
not an idea parameter name, imo. As far as I understand, it is used to bypass the save. So it would be better to name it according to what it does, instead of the single usecase that currently exists (e.g. bypassSave ou noSave)

Reply:
what about deleting a record in creation, is it correctly handled?